policy.go

```diff
@@ -36,9 +52,30 @@ var defaultPolicies = map[string]*policy{
 var activePolicies = make(policies)
 
 func (p policies) Get(key string) *policy {
+	// First look for an exact match
 	if pol, ok := p[key]; ok {
 		return pol
-	} else if pol, ok := p["*"]; ok {
+	}
+
+	// Now we're going to check for globs
+	// Order the keys in descending order of length
+	// so that "foobar*" takes precedence over "foo*"
+	// Now organize the keys by length
+	policyKeys := make(policyKeyList, 0, )
```

Comment:
Apparently it does, although I didn't mean to leave that comma in.  I took it out.